external/loaders/loaders/batches/_batch.py

```diff
@@ -89,6 +92,8 @@ def __post_init__(self):
                 "Timesteps provided for selection must be unique. "
                 f"Duplicated times were found: {duplicate_times}"
             )
+        if "ne" in self.res:
+            self.gsrm_name = "scream"
```

Comment:
Why provide `gsrm_name` as a config option if we just overwrite it based on `res`? If we can just use the existing `res` parameter to describe the horizontal grid, that would be better.

Reply:
We provide `gsrm_name` inside `mapper_config`. This is because functions like `open_nudge_to_fine` does not know anything about resolution but does need the name to read in the correct physics tendency variables. 
Then in the general config, instead of adding another `gsrm_name`, we use `res` to determine `gsrm_name`.